src/standalone/Vegetation/PlantHydraulics.jl

```diff
@@ -181,15 +189,21 @@ option (intendend only for debugging) to use a prescribed transpiration rate.
 
 $(DocStringExtensions.FIELDS)
 """
-struct PlantHydraulicsModel{FT, PS, T} <: AbstractPlantHydraulicsModel{FT}
+struct PlantHydraulicsModel{
+    FT,
+    PS,
+    T,
+    AA1 <: AbstractArray{FT},
```

Comment:
Im curious if AA1 and AA2 are both needed, if they are the same type (same length, element type)?

Reply:
No, if we can assume that they are exactly same length and type, they don't have to be distinct. I consolidated to one.